.github/workflows/run-integration-tests-basic.yml

```diff
@@ -0,0 +1,77 @@
+name: Minikube Integration Tests - basic
+on:
+   pull_request:
+      branches:
+         - master
+         - main
+         - 'releases/*'
+   push:
+      branches:
+         - master
+         - main
+         - 'releases/*'
+   workflow_dispatch:
+
+jobs:
+   run-integration-test:
+      name: Run Minikube Integration Tests
+      runs-on: ubuntu-latest
+      env:
+         KUBECONFIG: /home/runner/.kube/config
+      steps:
+         - uses: actions/checkout@v2
+
+         - name: Install dependencies
+           run: |
+              rm -rf node_modules/
+              npm install
+
+         - name: Install ncc
+           run: npm i -g @vercel/ncc
+         - name: Build
+           run: ncc build src/run.ts -o lib
+
+         - name: Set name of ns
```

Comment:
Even if we don't switch to the matrix yet it makes sense to make this the env variable. It's just more flexible that way and a more common practice. Setting it in an env variable is the right way to do it vs setting it in a step is more of a hack.